cmd/ingest/ingest.go

```diff
@@ -201,7 +201,7 @@ func main() {
 			log.Debugln("Mark as archived")
 			fileInfo := database.FileInfo{}
 			fileInfo.Path = archivedFile
-			fileInfo.Size = fileSize
+			fileInfo.Size, err := archive.GetFileSize(archivedFile)
```

Comment:
```suggestion
			fileInfo.Size, err := archive.GetFileSize(archivedFile)
			if err != nil {
				log.Errorf("Failed to get file size of: %s, reason: %v", archivedFile, err)
				continue
			}
```

Reply:
my editor complains about `:=` so `=` seems to work